packages/@react-types/color/src/index.d.ts

```diff
@@ -76,7 +76,7 @@ export interface Color {
   formatChannelValue(channel: ColorChannel, locale: string): string
 }
 
-export interface ColorFieldProps extends ValueBase<string | Color>, InputBase, Validation, FocusableProps, TextInputBase, LabelableProps {
+export interface ColorFieldProps extends Omit<ValueBase<string | Color>, 'onChange'>, InputBase, Validation, FocusableProps, TextInputBase, LabelableProps {
```

Comment:
Doesn't this get overridden below anyway? Do we need to explicitly omit it?

Reply:
@devongovett
You are right. However, when the [`--strict`](https://www.typescriptlang.org/tsconfig#strict) or [`--strictFunctionTypes`](https://www.typescriptlang.org/tsconfig#strictFunctionTypes) option is enabled, Typescript complains about this as you can see in the [issue](https://github.com/adobe/react-spectrum/issues/1666). (Maybe the issue creator uses IDE default TS option)

If we don't need to consider it now, It's better to close it now :)